### PR TITLE
✨Add product page for Clinical Data Flow

### LINKED ISSUE #12: Document clinical data flow product
Document the work needed to prototype and evaluate FHIR for clinical data distribution

### PR DESCRIPTION
**Pull request by znatty22**
Closes #12

### CODE REVIEW

File: docs/products/fhir-cdd.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+### [FHIR Model](https://github.com/kids-first/kf-fhir-model)
+
+**Issue Backlog:** [GitHub](https://github.com/kids-first/kf-model-fhir/issues)
+
+The repository containing the files for the Kids First FHIR data model along
+with CI and tooling for model validation and documentation.
+
+### [FHIR Data Service](https://github.com/kids-first/kf-model-fhir/issues)
+
+**Admin Dashboard:** http://10.10.1.191:9100
```
Comment (dankolbman) [inline docs/products/fhir-cdd.md]: Could you add a note that these are only available when on the CHOP network?

File: docs/products/fhir-cdd.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+## About
+
+D3b is responsible for coordinating a multitude of biobanking projects that
+aim to provide researchers with valuable biospecimens and data for rare
+diseases.
+
+Each of these projects has both common and unique requirements for
+clinical data collection, modeling, sharing, and search. It is
+infeasible to create a new data model and dataservice for each project.
```
Comment (allisonheath) [inline docs/products/fhir-cdd.md]: Well, it's probably feasible, but it's inefficient and also hinders interoperability/cross project analysis?

Comment (znatty22) [reply docs/products/fhir-cdd.md]: 👍

File: docs/products/fhir-cdd.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+  to reduce number of unique source data transformations
+
+## Overview:
+![Architecture](assets/clin-data-flow.png)
+
+## Organization
+
+### [Sprint Log](https://docs.google.com/document/d/1bkSWAT7BbLYdAgu_xyykALRJrdERIAn-9ae8t3HpQGI/edit?usp=sharing)
+
+The is kept on Google Docs and updated in preparation for the bi-weekly
```
Comment (dankolbman) [inline docs/products/fhir-cdd.md]: The *Sprint Log* is kept...

Comment (dankolbman) [reply docs/products/fhir-cdd.md]: I may have made this typo...

File: docs/products/cdf.md [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+control reports, and configure ETLs into the FHIR Data Service.
+
+### [FHIR Model](https://github.com/kids-first/kf-model-fhir)
+
+**Issue Backlog:** [GitHub](https://github.com/kids-first/kf-model-fhir/issues)
+
+The repository containing the files for the Kids First FHIR data model along
+with CI and tooling for model validation and documentation.
+
+### [FHIR Data Service](https://github.com/kids-first/kf-api-fhir-service/issues)
```
Comment (allisonheath) [inline docs/products/cdf.md]: Did you mean to link directly to issues? Or the top level repo?

**Comment by znatty22** [comment]
> Looks good to me, although the 10.0.0.0 addresses aren't returning anything?
> 🎳

Its probably bc you're not tunneling through dev bastion host. I think I'm going to remove those links because its confusing. They are all list on the repo README with instructions to tunnel anyway

File: docs/products/cdf.md (hunk 4 above)
Comment (znatty22) [reply docs/products/cdf.md]: Top level repo. will fix 👍

**Pull request merged**